mmv1/products/vertexai/terraform.yaml

```diff
@@ -34,6 +35,28 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_from_api: true
       region: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
+  # Vertex AI Endpoint
+  Endpoint: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: false
+    skip_sweeper: true
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 6
+      update_minutes: 6
+      delete_minutes: 10
+    examples:
```

Comment:
We'll want to cover every new field introduced using examples, as well as include handwritten tests to cover updating the resource. Covering every updatable field would be excellent, but covering a few as a baseline works for resource w/ a single update method.